packages/app-store/routing-forms/lib/handleResponse.ts

```diff
@@ -185,25 +152,29 @@ const _handleResponse = async ({
     } else {
       // It currently happens for a Router route. Such a route id isn't present in the form.routes
     }
-
-    let dbFormResponse;
+    let dbFormResponse, queuedFormResponse;
     if (!isPreview) {
-      dbFormResponse = await prisma.app_RoutingForms_FormResponse.create({
-        data: {
-          // TODO: Why do we not save formFillerId available in the input?
-          // formFillerId,
+      if (queueFormResponse) {
```

Comment:
Isn't this always False? we declare queuedFormResponse as undefined in L155 and we never modify the var until this conditional

Reply:
L157 is looking for the param `queueFormResponse` while L155 is declaring the variables for the `queuedFormResponse`